### ISSUE TITLE
CI broken because of the installation of Why3 via OPAM

### ISSUE THREAD

**Issue by Niols** [colis-anr/colis-language#118]
The CI is now broken because of a failure in the installation of Why3 via OPAM.

```
#=== ERROR while compiling why3.dev ===========================================#
# context              2.0.6 | linux/x86_64 | ocaml-base-compiler.4.07.1 | pinned(git+https://gitlab.inria.fr/why3/why3.git#4fbafaac#4fbafaac)
# path                 ~/.opam/4.07/.opam-switch/build/why3.dev
# command              /usr/bin/make -j71 all opt byte
# exit-code            2
# env-file             ~/.opam/log/why3-7-637bd5.env
# output-file          ~/.opam/log/why3-7-637bd5.out
### output ###
# [...]
# Ocamlc   src/session/xml.ml
# Ocamlopt src/tools/unix_scheduler.ml
# gcc -Wall -o lib/why3cpulimit src/server/cpulimit-unix.o src/server/cpulimit-win.o
# Ocamlopt src/tools/why3wc.ml
# Linking  bin/why3wc.byte
# Ocamlopt src/util/pp.ml
# Ocamlc   src/util/wstdlib.mli
# Ocamlopt src/util/cmdline.ml
# File "_none_", line 1:
# Error: Cannot find file menhirLib.cmo
# make: *** [Makefile:2291: bin/why3wc.byte] Error 2
# make: *** Waiting for unfinished jobs....
```

This is also what is happening on my personal machine with both the old pinned version and the master branch of Why3, and this no matter the version of OCaml. @benozol @claudemarche Do you have any idea?

**Comment by claudemarche** [comment]
Probably related to https://gitlab.inria.fr/why3/why3/issues/436, but that issue is closed so the problem should be solved in the master branch of Why3.
You may consider reopening that issue by detailed your exact configuration. But first you should check the menhir version you have. Maybe too many pinned versions in your opam ?

**Comment by Niols** [comment]
OK, then I guess it will be solved by #117 when it is merged.
